docs/make.bat

```diff
@@ -1,5 +1,19 @@
-@ECHO OFF
+REM Copyright 2018 Intel Corporation

```

Comment:
This file should be deleted entirely. We don't support building docs directly on Windows, and would use Docker with the Makefile instead in that environment.